packages/fxa-auth-server/lib/routes/utils/signin.js

```diff
@@ -403,11 +406,18 @@ module.exports = (log, config, customs, db, mailer) => {
           uid: accountRecord.uid,
         });
 
+        const secret = accountRecord.primaryEmail.emailCode;
+        const authenticator = new otplib.authenticator.Authenticator();
+        authenticator.options = Object.assign(
+          otplib.authenticator.options,
+          otpOptions,
+          { secret }
+        );
         const geoData = request.app.geo;
         return mailer
           .sendVerifyLoginCodeEmail(accountRecord.emails, accountRecord, {
             acceptLanguage: request.app.acceptLanguage,
-            code: sessionToken.tokenVerificationCode,
```

Comment:
Where else is tokenVerificationCode used? What I'm really asking is can https://github.com/mozilla/fxa/blob/913e67f6b483c1b7716906043e1f017ced401cfa/packages/fxa-auth-server/lib/routes/account.js#L205 and a lot of the supporting infrastructure be removed?